backend/small_eod/user/models.py

```diff
@@ -0,0 +1,15 @@
+from django.contrib.sessions.models import Session
+
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+from guardian.mixins import GuardianUserMixin
+
+
+class User(GuardianUserMixin, AbstractUser):
+    def __str__(self):
+        return self.username
+
+
+class Description(models.Model):
```

Comment:
Co to jest?

Reply:
model Description z dokumentacji. Niezbyt wiem po co on jest potrzebny, ale w swaggerze bylo polaczenie z obiektem session - dalem go wiec do apki users, bardziej jako template. Moge prosic o wyjasnienie jaki byl zamysl przy dokumentowaniu tego obiektu?

Reply:
Zobacz na https://demo.small-eod.siecobywatelska.pl/admin/cases/lettername/ . To jest opis listu https://demo.small-eod.siecobywatelska.pl/admin/cases/letter/5963/change/ . Nie wiem gdzie masz powiązanie z ```Session``` do ```Description```.

Reply:
Niestety nie mam credentiali zeby sie zalogowac. Foreign Key z description do session stworzony.

Reply:
@silv3rmat , dostępne na https://watchdogsii.slack.com/archives/CHEG3L4SE/p1578367963009000 .

Reply:
> Foreign Key z description do session stworzony.
Chyba się zrozumieliśmy zupełnie odwrotnie.

Ja pisząc:
>  Nie wiem gdzie masz powiązanie z Session do Description.
odnosiłem się do:
> model Description z dokumentacji. (...) bylo polaczenie z obiektem session

Model "Description" powinien istnieć w "letters" i powinna istnieć relacja FK z "Letter" do "Description". Takie specyficzne, że listy nie mają opisów dowolnych, a z zamkniętego katalogu, bo to jest wyrok, skarga itp.

Reply:
```
Description{
id* | string($sessionId)readOnly: true
text* | string($name)maxLength: 200
userAgent | string
createdOn* | string($datetime)title: Czas utworzenia treścireadOnly: true
createdBy* | stringreadOnly: trueUser ID responsible for creating the content
modifiedOn* | string($datetime)title: Czas ostatniej modyfikacji treścireadOnly: true
modifiedBy* | stringtitle: User ID responsible for the latest content changereadOnly: true
}
```
Ja sie odnosilem do tego - ale rozumiem ze to bylo przegapione. 
Podsumowujac:
1. Wrzucic description do letter
2. Zrobic FK description -> letter
3. usunac pole UserAgent z description

Reply:
Tak, było przegapione. 34959a1082a679d83b34797878334195f9ca3c0c